src/schema/objects/entities.yaml

```diff
@@ -349,10 +349,13 @@ task:
   name: task
   display_name: Task
   description: |
-    Each task has a unique label that MUST only consist of letters and/or
-    numbers (other characters, including spaces and underscores, are not
-    allowed).
```

Comment:
I believe this part of the rule section of the schema and I have therefore removed it even if it is in the common principles sections